Sources/Amplitude/Amplitude.m

```diff
@@ -1771,7 +1771,7 @@ - (id)unarchive:(NSString *)path {
 }
 
 - (id)unarchive:(NSData *)data error:(NSError **)error {
-    if (@available(iOS 12, tvOS 11.0, macOS 10.13, watchOS 4.0, *)) {
+    if (@available(iOS 11, tvOS 11.0, macOS 10.13, watchOS 4.0, *)) {
```

Comment:
Safe for 11 as well?

Reply:
Yep, see https://developer.apple.com/documentation/foundation/nskeyedarchiver/2962880-archiveddatawithrootobject and https://developer.apple.com/documentation/foundation/nskeyedunarchiver/2962884-unarchivedobjectofclass. It works in my local when building with iOS 11 too.